packages/workbox-background-sync/demo/sw.js

```diff
@@ -1,8 +1,8 @@
 /* global workbox*/
 importScripts(
-	'../build/background-sync-queue.js',
-	'../../workbox-routing/build/workbox-routing.js',
-	'../../workbox-runtime-caching/build/workbox-runtime-caching.js'
+	'../build/importScripts/workbox-background-sync.dev.v0.0.1.js',
+	'../../workbox-routing/build/importScripts/workbox-routing.dev.v0.0.1.js',
+	'../../workbox-runtime-caching/build/importScripts/workbox-runtime-caching.dev.v0.0.1.js'
```

Comment:
@gauntface Noob question: this path itself is longer than 80 chars, and fails lint how to fix this?